.env.sample

```diff
@@ -1,6 +1,5 @@
 REACT_APP_TERRAWARE_API=http://localhost:8080
-KEYCLOAK_RESOURCE=
-KEYCLOAK_CREDENTIALS_SECRET=
-KEYCLOAK_REALM=
-KEYCLOAK_AUTH_SERVER_URL=https://auth.staging.terraware.io
+SPRING_SECURITY_OAUTH2_CLIENT_PROVIDER_KEYCLOAK_ISSUERURI=https://auth.staging.terraware.io/realms/terraware
+SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_KEYCLOAK_CLIENTSECRET=
```

Comment:
missed this.. do we need a `SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_CLIENTID`?

Reply:
The client ID defaults to `dev-terraware-server`.